ee/api/ee_event_definition.py

```diff
@@ -21,6 +22,9 @@ class Meta:
             "updated_at",
             "updated_by",
             "last_seen_at",
+            "verified",
```

Comment:
Seems like `verified` is redundant if we have `verified_at`, but unsure if we'd plan to eventually get rid of that attribute with the version control history.

Reply:
Yup that's what I had planned as well. `verified_at` will be eventually deprecated once version control becomes a thing.